style.css

```diff
@@ -0,0 +1,80 @@
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0% 37%;
+}
+
+header {
+  height: 92px;
+  width: 355px;
+  background-color: #3c3a39;
+  padding: 10px;
+  color: white;
+}
```

Comment:
[OPTIONAL] I think it is a good idea to reduce the height property of the header. :smiley: :smiley: 
![image](https://user-images.githubusercontent.com/98363075/192077144-59ccf1f7-66dd-493d-a50d-0aefeb929aa2.png) 
![image](https://user-images.githubusercontent.com/98363075/192077163-9ee41785-8954-4275-b973-e5f6d2c5af03.png)